protostar/commands/test/cheatcodes/skip_cheatcode.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Callable
+
+from protostar.starknet.cheatcode import Cheatcode
+
+from protostar.commands.test.test_environment_exceptions import (
+    SimpleSkippingReportedException,
+)
+
+
+class SkipCheatcode(Cheatcode):
+    @property
+    def name(self) -> str:
+        return "skip"
+
+    def build(self) -> Callable[..., Any]:
+        return self.skip
+
+    def skip(self, condition: bool = True, reason: str = "") -> None:
+        if condition:
```

Comment:
i don't know if i like that `condition` thing... i'll let others decide on that one, but i have mixed feelings about this, since one can just use `if` inside of the hint.

Reply:
Hmm, using the `if` is actually neat!

Pytest doesn't have such opportunity because `skip` is a decorator there. I agree with @Arcticae